charts/kyverno/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: cray-kyverno
-version: 1.5.0
+version: 1.3.2
```

Comment:
Should this be 1.5.1?

Reply:
When we fix in main it should be 1.5.1 and for 1.4 which we fix recently we had 1.4.1. Since this is fix for 1.3 (current version 1.31) it should be 1.3.2.

Reply:
I ask because Jeremy just bumped this up to 1.5 in the csm-1.3 branch a few weeks ago.   This seems to undo that.

Reply:
The source branch for this PR is based on main. From a content perspective, I'm going to diff what we have tagged in CSM 1.3 (and 1.3.1) and circle back.

Reply:
Confirmed this should be 1.3.2. Confirmed the set of changes incoming via this PR as for CASMPET-6398 and CASMSEC-385 (along for the ride, but good content).